src/System.Linq.Expressions/src/System/Linq/Expressions/Error.cs

```diff
@@ -872,7 +881,7 @@ internal static Exception InstancePropertyWithoutParameterNotDefinedForType(obje
         /// </summary>
         internal static Exception InstancePropertyWithSpecifiedParametersNotDefinedForType(object p0, object p1, object p2)
         {
-            return new ArgumentException(Strings.InstancePropertyWithSpecifiedParametersNotDefinedForType(p0, p1, p2));
+            return new ArgumentException(Strings.InstancePropertyWithSpecifiedParametersNotDefinedForType(p0, p1, p2), "propertyName");
```

Comment:
I think I will,  thanks. The downside is that it adds to the size of the calling method, but since the only caller has a large number of similar calls in it that downside isn't much here. It would be another matter if the caller was very small.